port/common/omrsysinfo.c

```diff
@@ -163,6 +163,40 @@ omrsysinfo_get_processor_feature_name(struct OMRPortLibrary *portLibrary, uint32
 	return rc;
 }
 
+/**
+ * Generate the corresponding string literals for the provided OMRProcessorDesc.
+ *
+ * @param[in] portLibrary The port library.
+ * @param[out] buff The cpu features output string.
+ * @param[in] BUFF_SIZE The size of the buff.
+ */
+void
+omrsysinfo_print_processor_features(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc, char * buff, const size_t BUFF_SIZE)
+{
+	memset(buff, 0, BUFF_SIZE*sizeof(char));
+	size_t rowLength = 0;
```

Comment:
This variable name is misleading. It seems to be used as a write cursor and should be named accordingly. There are also no rows as it is all written in one line. I think we should re-adapt the name.